src/main.py

```diff
@@ -87,22 +88,40 @@ def _enable_kolibri_plugin(plugin_name: str, optional=False) -> bool:
 except FileNotFoundError:
     pass
 
-# we need to initialize Kolibri to allow us to access the app key
-initialize()
 
-# start kolibri server
-logging.info("Starting kolibri server via Android service...")
-start_service("server")
+def start_kolibri():
+    # we need to initialize Kolibri to allow us to access the app key
+    initialize()
 
-# Tie up this thread until the server is running
-wait_for_status(STATUS_RUNNING, timeout=120)
+    # start kolibri server
+    logging.info("Starting kolibri server via Android service...")
+    start_service("server")
 
-_, port, _, _ = _read_pid_file(PID_FILE)
+    # Tie up this thread until the server is running
+    wait_for_status(STATUS_RUNNING, timeout=120)
 
-start_url = "http://127.0.0.1:{port}".format(port=port) + interface.get_initialize_url()
-loadUrl(start_url)
+    _, port, _, _ = _read_pid_file(PID_FILE)
 
-start_service("remoteshell")
+    start_url = (
+        "http://127.0.0.1:{port}".format(port=port) + interface.get_initialize_url()
+    )
+    loadUrl(start_url)
+
+    start_service("remoteshell")
```

Comment:
This bit is unrelated to Kolibri itself. We should start this service earlier, and ideally somewhere else just so it's clearer.